### DISCUSSION TITLE
STF IN WIFI NETWORKS

### DISCUSSION THREAD

**Discussion by RobsonBreno** [DeviceFarmer/stf#589] [General]
**Is it possible for this project to work on external networks? wifi 3g 4g 5g ? Would be great .**

**Comment by denis99999** [comment]
@RobsonBreno , yes it is possible, but it does not depend on STF, or specify a little more your thought?

**Reply by RobsonBreno** [reply]
Friend, contact me, I have a proposal for you. My skype ( live:.cid.f1f80332caf961bd )
